packages/haproxy/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -157,11 +157,11 @@ processors:
   - append:
       field: event.category
       value: network
-      if: "ctx?.source.ip != null && ctx?.destination?.ip != null"
+      if: ctx.source?.ip != null && ctx.destination?.ip != null
```

```suggestion
      if: ctx?.source?.ip != null && ctx?.destination?.ip != null
```

Slightly more fail safe. Adding null check for both ctx and source/destination

---

@ishleenk17 `ctx` can never be null in this context, therefore it is safe to remove the null-safe operators.

This removal was done at my ask in https://github.com/elastic/integrations/pull/11594#discussion_r1823237862